public/app/features/alerting/unified/api/timeIntervalsApi.ts

```diff
@@ -7,7 +7,7 @@
 // which may help alleviate this when it lands:
 // https://github.com/reduxjs/redux-toolkit/pull/3485
 
-import { DefinitionsFromApi, OverrideResultType } from '@reduxjs/toolkit/dist/query/endpointDefinitions';
+import { DefinitionsFromApi, OverrideResultType } from '@reduxjs/toolkit/query';
```

Comment:
(The offending imports are gone in https://github.com/grafana/grafana/pull/90094, but fixing now instead of waiting for that to land)